Userland/Libraries/LibWeb/CSS/Default.css

```diff
@@ -64,6 +64,22 @@ button:hover, input[type=submit]:hover, input[type=button]:hover, input[type=res
     background-color: -libweb-palette-hover-highlight;
 }
 
+input[type=checkbox] {
+    display: inline-block;
+    width: 12px;
+    height: 12px;
+    margin: 2px;
+    border: 1px solid -libweb-palette-threed-shadow1;
+}
+
+input[type=checkbox]:checked {
+    /* This roughly resembles ClassicStylePainter's paint_check_box() while uncoupling the styling from LibGfx,
+       similar to <button> above. This is a simple checkmark that still looks ok at 2x scale, eventually we
```

Comment:
nit: replace `,` with `.` as the part after it is unrelated to the part before it

Reply:
Done, slightly expanded the comment, and credited Xexxa. Thanks!